_rest_routes.inc.php

```diff
@@ -757,14 +769,30 @@
         return $return;
     },
     "GET /fhir/Observation" => function (HttpRestRequest $request) {
-        RestConfig::authorization_check("patients", "med");
-        $return = (new FhirObservationRestController())->getAll($_GET);
+        if ($request->isPatientRequest()) {
+            $getParams = $_GET;
+            // TODO: need to verify that observation rest controller supports search by patient.
+            $getParams['patient'] = $request->getPatientUUIDString();
+            $return = (new FhirObservationRestController())->getAll($getParams);
+        } else {
+            RestConfig::authorization_check("patients", "med");
+            $return = (new FhirObservationRestController())->getAll($_GET);
+        }
+
         RestConfig::apiLog($return);
         return $return;
     },
     "GET /fhir/Observation/:uuid" => function ($uuid, HttpRestRequest $request) {
-        RestConfig::authorization_check("patients", "med");
-        $return = (new FhirObservationRestController())->getOne($uuid);
+        if ($request->isPatientRequest()) {
```

Comment:
Will need to make this endpoint patient request proof. I'll add this to my queue.
TODO for me

Reply:
this is almost completed. i'll plan to bring in that separate PR after this PR is in codebase